charts/portworx/v1.0.0/Chart.yaml

```diff
@@ -0,0 +1,21 @@
+name: portworx
+appVersion: 1.0.0
+version: 1.0.0
+description: A Helm chart for installing Portworx on Kubernetes.
+tillerversion: ">=2.9.0"
```

Comment:
need to remove the tillerVersion as rancher uses a custom release, and the version is up-to-the-date using v2.10.0.